src/bci_build/package/base/README.md.j2

```diff
@@ -0,0 +1,30 @@
+# SLE BCI {{ image.os_version.pretty_os_version_no_dash }} Base Container Image
+{% include 'badges.j2' %}
+
+## Description
+
+SUSE Linux Enterprise Base Container Images (SLE BCI) provide truly open,
+flexible, and secure container images and application development tools. The
+images consist of container environments based on SUSE Linux Enterprise and
+designed to be a secure base for any containerized workload.
+
+SLE BCI is freely available, re-distributable, and supported across many
+different environments. These templates and tools address modern, containerized
+application development and CI/CD application containerization.  They can be
+used immediately by developers and integrators without the “lock-in” imposed by
+other offerings.
+
+SLE BCI inherits industry-leading security and compliance from SUSE Linux
+within your container build process. The container images are designed to be a
+secure base for any application workload. SUSE ensures that compliance
+standards are applied consistently and continuously improves security-related
+capabilities.
+
+SLE BCI is lightweight and easy to adopt, with the ability to run with any
+Linux OS. Avoid lock-in imposed by other vendors and get exactly what you need,
```

Comment:
nit: it would be nice to make it clear that Linux OS here is the host OS of the container runtime, otherwise it gives an impression that you can run BCI with another OS inside.

Reply:
I'd like to address this in a followup. this one is intended to as closely as possible resemble the existing sle15-image. we can clean it up once this PR has been merged. it is large enough as is.